docs/content/latest/integrations/spring-framework/sdyb.md

```diff
@@ -0,0 +1,141 @@
+    <groupId>com.yugabyte</groupId>
+    <artifactId>spring-data-yugabytedb-ysql</artifactId>
+    <version>2.3.0</version>
+</dependency>
+<dependency>
+  <groupId>com.yugabyte</groupId>
+  <artifactId>jdbc-yugabytedb</artifactId>
+  <version>42.2.7-yb-5.beta.1</version>
+</dependency>
+```
+
+## Examples
+
+
+The following example demonstrates how to create a basic shopping cart using SDYB:
+
+```java
+public interface ShoppingCartRepository extends YsqlRepository<ShoppingCart, String> {
+  
+  ShoppingCart findById(String id);
+  List<ShoppingCart> findByUserId(String userId);
+  
+}
+
+@Service
+public class CartService {
+
+  private final ShoppingCartRepository repository;
+  
+  public CartService(CartService repository) {
+    this.repository = repository;
+  }
+
+  public void doWork() {
+    repository.deleteAll();
+    
+    ShoppingCart myShoppingCart = new ShoppingCart();
+    myShoppingCart.set("cart1")
+    myShoppingCart.setUserId("u1001");
+    myShoppingCart.setProductId("asin1001");
+    myShoppingCart.setQuantity(1);
+
+    repository.save(myShoppingCart);
+
+    ShoppingCart savedCart = repository.findById("cart1");
+
+    List<ShoppingCart> productsInCart = repository.findByUserId("u1001");
+    repository.count();
+  }
+
+}
+
+@Configuration
+@EnableYsqlRepositories
+public class YsqlConfig extends AbstractYugabyteJdbcConfiguration {
+
+  @Bean
+  DataSource dataSource() {
+    String hostName = "127.0.0.1";
+    String port = "5433";
+
+    Properties poolProperties = new Properties();
+    poolProperties.setProperty("dataSourceClassName",
+    "com.yugabyte.ysql.YBClusterAwareDataSource");
+    poolProperties.setProperty("dataSource.serverName", hostName);
+    poolProperties.setProperty("dataSource.portNumber", port);
+    poolProperties.setProperty("dataSource.user", "yugabyte");
+    poolProperties.setProperty("dataSource.password", "");
+    poolProperties.setProperty("dataSource.loadBalance", "true");
+    poolProperties.setProperty("dataSource.additionalEndpoints",
+    "127.0.0.2:5433,127.0.0.3:5433");
+
+    HikariConfig hikariConfig = new HikariConfig(poolProperties);
+    DataSource ybClusterAwareDataSource = new HikariDataSource(hikariConfig);
+    return ybClusterAwareDataSource;
+  }
+
+  @Bean
+  JdbcTemplate jdbcTemplate(@Autowired DataSource dataSource) {
+
+    JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
+    return jdbcTemplate;
+  }
+
+  @Bean
+  NamedParameterJdbcOperations namedParameterJdbcOperations(DataSource dataSource) { 
+    return new NamedParameterJdbcTemplate(dataSource);
+  }
+
+  @Bean
+  TransactionManager transactionManager(DataSource dataSource) {           
+    return new YugabyteTransactionManager(dataSource);
+  }
+
+}
+```
+
+## Compatibility Matrix
+
+| **SDYB** | **YugabyteDB Smart Driver** | **Postgres JDBC Driver** |
+| -------- | --------------------------- | ------------------------ |
+| 2.3.0    | 42.2.7-yb-5.beta.1          | 42.2.7 or later          |
+
```

Comment:
Please add the following line at the end 

```
To learn more about using the Spring Data YugabyteDB module, see[ Spring Data YugabyteDB example](https://github.com/yugabyte/spring-data-yugabytedb-example).
```

Reply:
Added at the end of "Examples".